wordpress.org/public_html/wp-content/themes/pub/wporg-developer/inc/user-content.php

```diff
@@ -458,6 +458,8 @@ public static function get_editor_rules( $context = '', $note_link = '' ) {
 				. $feedback_rule
 				. '</li><li>'
 				. __( 'This form is not for support requests, spam, bug reports, complaints, or self-promotion. Entries of this nature will be deleted.', 'wporg' )
+				. '</li><li>'
+				. __( 'Notes and feedback must be written in English. Entries not written in English will be deleted.', 'wporg' )
```

Comment:
Hey @crstauf I think it'd be better if we updated the language at the end to read

`Entries not written in English will not be accepted`

Reply:
@Kenshino thanks!

is there an alternative that doesn’t use a double negative, to make comprehension easier?

i’m not sure if it’s an actual grammar rule, but i try to avoid using them.

Reply:
Perhaps another way to write this would be to remove the "will be deleted" from the existing rules and this new added one, and a new entry added at the end along the lines of "Any notes not meeting these requirements will be removed by the moderation team."

Reply:
I like @dd32 's suggestion :)

Reply:
Adjusted in 694cbdb.